### DISCUSSION TITLE
How to use AIO's TURN proxy for all network traffic.

### DISCUSSION THREAD

**Discussion by c469591** [nextcloud/all-in-one#3379] [Questions] [answered]
Hello, I saw in the description that AIO includes TURN service. 
I have successfully deployed AIO now and Nextcloud Talk is working smoothly. 
I would like to know how to route all traffic of Nextcloud Talk through the server without using any P2P connections. 
I didn't see any TURN installation options in the aio setup webpage.
Thank you!

**Comment by szaimen** [comment]
Hi, I am not completely sure what you want to achieve and why. Can you please paraphrase and explain your request more in detail? Thank you!

**Comment by c469591** [comment]
Hello,
What I mean is to let my nextcloud talk forward all the data of voice calls or other chats through the server, instead of using peer-to-peer connections. 
Because I found that in some areas, peer-to-peer connections are not very stable, and my server happens to be located in an area where both parties can connect smoothly. That's why I want to hand over all the traffic to the server as a relay server.
Thank you!

**Comment by szaimen** [comment, marked as answer]
This would be something to ask for in the nextcloud/spreed app then :)

**Comment by c469591** [comment]
How can I force all transmissions to go through the server instead of using peer-to-peer transmission?
